src/edit/main.js

```diff
@@ -512,6 +512,15 @@ export class ProseMirror {
   markAllDirty() {
     this.dirtyNodes.set(this.doc, DIRTY_REDRAW)
   }
+
+  // :: (string) → string
+  // Return a translated string, if a translate function has been supplied,
+  // or the original string.
+  translate(string) {
```

Comment:
Is there any existing convention for a short name to use for this? I expect that in some types of code it'll show up a lot. `l10n` might be a bit too obscure.